src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,76 @@
+package duke.task;
+
+/**
+ * Abstract class contains variables and methods related to Tasks.
+ */
+public abstract class Task {
+    protected boolean completed;
```

Comment:
could name it isCompleted, so it sounds more like a boolean!

Reply:
I'll change that, thank you!